src/utilities/identities/identities.ts

```diff
@@ -267,7 +274,13 @@ export async function getIdentityCryptoFromKeypair(
 
   function sign(plaintext: string) {
     const signature = Crypto.u8aToHex(authenticationKey.sign(plaintext));
-    const didKeyUri = didDetails.getKeyIds(KeyRelationship.authentication)[0];
+
+    const authorizationKey = didDetails.getVerificationKeys(
```

Comment:
Not sure, but `IDidDetails` could be `DidDetails`, and this could then be `didDetails.authenticationKey`.